src/soilbiogeochem/SoilBiogeochemDecompCascadeMIMICSMod.F90

```diff
@@ -0,0 +1,1024 @@
+            !        mean) or t_mo_patch (30-d avg) or ...
+            vmax_l1_m1 = exp(vslope_l1_m1 * t_soi24_degC_col(c,j) + vint_l1_m1) * vmod_l1_m1
+            vmax_l1_m2 = exp(vslope_l1_m2 * t_soi24_degC_col(c,j) + vint_l1_m2) * vmod_l1_m2
+            vmax_l2_m1 = exp(vslope_l2_m1 * t_soi24_degC_col(c,j) + vint_l2_m1) * vmod_l2_m1
+            vmax_l2_m2 = exp(vslope_l2_m2 * t_soi24_degC_col(c,j) + vint_l2_m2) * vmod_l2_m2
+            vmax_s3_m1 = exp(vslope_s3_m1 * t_soi24_degC_col(c,j) + vint_s3_m1) * vmod_l2_m1
+            vmax_s3_m2 = exp(vslope_s3_m2 * t_soi24_degC_col(c,j) + vint_s3_m2) * vmod_l2_m2
+
+            ! km are time-dependent terms
+            km_l1_m1 = exp(kslope_l1_m1 * t_soi24_degC_col(c,j) + kint_l1_m1) * kmod_l1_m1
+            km_l1_m2 = exp(kslope_l1_m2 * t_soi24_degC_col(c,j) + kint_l1_m2) * kmod_l1_m2
+            km_l2_m1 = exp(kslope_l2_m1 * t_soi24_degC_col(c,j) + kint_l2_m1) * kmod_l2_m1
+            km_l2_m2 = exp(kslope_l2_m2 * t_soi24_degC_col(c,j) + kint_l2_m2) * kmod_l2_m2
+            km_s3_m1 = exp(kslope_s3_m1 * t_soi24_degC_col(c,j) + kint_s3_m1) * kmod_s3_m1 / p_scalar(c,j)
+            km_s3_m2 = exp(kslope_s3_m2 * t_soi24_degC_col(c,j) + kint_s3_m2) * kmod_s3_m2 / p_scalar(c,j)
+
+            ! Product of w_scalar * depth_scalar * o_scalar
+            w_d_o_scalars = w_scalar(c,j) * depth_scalar(c,j) * o_scalar(c,j)
+
+            ! decomp_k used in SoilBiogeochemPotentialMod.F90
+            ! TODO Defined correctly here?
+            ! TODO Combine multiple terms of same donor pools...
+            ! TODO Where to calculate m1_pool and m2_pool?
+            ! TODO When to introduce litter/soil biomass to the calculation?
+            ! TODO Excluded spinup terms for now. See cwd below how used.
+            decomp_k(c,j,i_met_lit) = (vmax_l1_m1 * m1_pool(c,j) / &
+                                       (km_l1_m1 + m1_pool(c,j))) * &
+                                      w_d_o_scalars
+            decomp_k(c,j,i_met_lit) = (vmax_l1_m2 * m2_pool(c,j) / &
+                                       (km_l1_m2 + m2_pool(c,j))) * &
+                                      w_d_o_scalars
+
+            decomp_k(c,j,i_str_lit) = (vmax_l2_m1 * m1_pool(c,j) / &
+                                       (km_l2_m1 + m1_pool(c,j))) * &
+                                      w_d_o_scalars
+            decomp_k(c,j,i_str_lit) = (vmax_l2_m2 * m2_pool(c,j) / &
+                                       (km_l2_m2 + m2_pool(c,j))) * &
+                                      w_d_o_scalars
+
+            decomp_k(c,j,i_pas_som) = (vmax_s3_m1 * m1_pool(c,j) / &
+                                       (km_s3_m1 + m1_pool(c,j))) * &
+                                      w_d_o_scalars
+            decomp_k(c,j,i_pas_som) = (vmax_s3_m2 * m2_pool(c,j) / &
+                                       (km_s3_m2 + m2_pool(c,j))) * &
+                                      w_d_o_scalars
```

Comment:
these two should be i_avl_som
and *_s1_m*
all references to ACT_SOM should be AVL_SOM which is S1

Reply:
again sum the two for decom_k of i_avl_som